### DISCUSSION TITLE
Genesis Gov.epoch proposal

### DISCUSSION THREAD

**Discussion by DanM3rcurius** [SecretDecks/Documentation#80] [gov.channel]
### Background 
 I was looking at the mini proposals done by Tevo from Catalyst Swarm and thought about what if we employ this for executive level/high impact decisions: 
#### for example
I was thinking to do a pre sale of a limited number of 3 card packs. These packs would be closed and covered in a slick animated artwork, available in 3 different rarity types. Once the drop of the first Deck happens anyone with the pack can redeem it for 3 pulls from the drop.
To greenlight this, is an **executive decision** that can have a huge impact, so in order to vote on sth like this my idea was to put up a mini proposal describing what change/decision needs to be done.

### Problem 
We don't have funding and it's still too early to apply for Catalyst funding in my mind (or is it?!), we have many people working on it now and small running expenses (server and domains) that we need to reimburse. 

### Possible Solution
Do a "community founders" pre sale. With 333 packs (3 different rarity types) to be sold for 78 ADA each. A pack itself is an NFT artwork and enables holder to draw 3 (?) tarot cards in 1st drop. That way we can give back to the artists creating the Pack artwork/animation, cover the running costs for a long time and reward contributors via our tracked contributions data; perfect test bed for what we want to do

### Requirements
- (unanimous) yes vote from core
- animated artwork
- pre sale page setup [coop with Kyle (from Adosia), Quinn (TAC) or Patrick Tobler]

Thoughts @SecretDecks/founding-core ?

**Comment by stephen-rowan** [comment]
Apply now - the funding will not arrive until November. That gives Secret Decks a milestone to work to. If we get funded we can always re-negotiate our roadmap with Project Catalyst.

**Comment by wlstylr** [comment]
1.) I agree that we should start thinking about a drop as you described, and take a vote to possibly initiate the progress.

2.) Have you seen Patrick's latest offering? https://www.nft-maker.io/pro

3.) Depending on the current price of ADA, we may need to decrease the cost of a pack. 

4.) We should apply now, but the proposal should be for a piece of functionality that will be useful to many, not a proposal for an NFT drop.

**Reply by DanM3rcurius** [reply]
2. Yes have seen it, was checking fo a friend who minted some NFTs there. We could use him as a backend or do our own, depending on workload of @tamagit80 and @bUxEE 

3. agreed, the price should be realistic and adapted shortly before drop

4. I wrote a proposal draft #81 , I filled in fields on ideascale and sent screenshots into the discord for approval!

**Comment by evolayak** [comment]
Yeah, I agree to start this. I'd like to get a better understanding what capacity we have for animation (I'm only starting to learn aftereffects and refining my cropping/masking skills on photoshop :D); what decks are ready to be animated, etc.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Reply by DanM3rcurius** [reply]
During the pre-sale we would sell only the unopened packs, not the cards themselves.
So anyone can can go wild with the SD-logo and their imagination!
In case we won't find artists in the next 2 weeks that commit to the project and collaborate with us, we'll go ahead and use the Rider Waite deck as I originally intended.

**Comment by tamagit80** [comment]
Hello,

I agree with you guys. Let's apply. Backend to create NFT is working
progress, not easy but we can do it ;) talked yesterday with Alberto and
probably middle/end september we will have our system.
About the decks, I'll call Tenia and Ambra and ask them the original files.
They agreed. Animations... whenever u want, installed after effect on my
computer... sadly! Hate Adobe buuuuut... we can do it easy!

ciao have a nice beginning of the week!

**Reply by DanM3rcurius** [reply]
Good evening @tamagit80 and thanks for tuning in 🙏🏿 
Amazing news! lets get it done, maybe a power session at your place to come up with the package design? 
We can even work without T&A's designs and go more our brand (even as simple as just the logo, claim & smoke effect from the website!), but ofc working with theirs for the package design would be more amazing!
Once you have the files, we need to do multiple brainstorm sessions on how to animate each card and then see who we can find to help us, or do it ourselves 💪🏿  (ie Heather and you^^ we could ask Jonathan if he wants to help?)

Thoughts?

**Comment by tamagit80** [comment]
Sure would be amazing. Full immersion. There is also a new team member for
ux/ui. She will join us. Let me know when u have time

**Reply by DanM3rcurius** [reply]
Terrific! reinforcements🪂 in 😄   
First week of September would be best for me, then I can free up a day and a night most likely!
